src/main/java/creativei/service/impl/InquiryServiceImpl.java

```diff
@@ -61,7 +67,7 @@ public Inquiry create(Inquiry inquiry) throws UniqueConstraintViolationException
             if(de.getCause() instanceof ConstraintViolationException){
                 ConstraintViolationException ce = (ConstraintViolationException) de.getCause();
                 if(ce.getConstraintName()==null)
-                    throw new InvalidParamRequest("Required Field Can not be Empty");
+                    throw new InvalidParamRequest(ce.getCause().getMessage());
```

Comment:
Log this error. Use more generic error message in response

Reply:
sir logger is there when exception is catched there is logger . Review it in commit d5326cd